src/vos/vos_aggregate.c

```diff
@@ -2550,7 +2550,7 @@ aggregate_enter(struct vos_container *cont, int agg_mode, daos_epoch_range_t *ep
 		}
 
 		if (cont->vc_in_aggregation) {
-			D_ERROR(DF_CONT": In aggregation epr["DF_U64", "DF_U64"]\n",
+			D_DEBUG(DB_EPC, DF_CONT": In aggregation epr["DF_U64", "DF_U64"]\n",
```

Comment:
This information is still useful even for production system. Though ERROR is indeed too much. Maybe it should only output after each certain interval. Maybe we should add

DL_INFO(interval, "xxxxx") ?

Reply:
To me, this should not be `INFO` because of the caller polling/retrying, so I would vote for `DEBUG`. @jolivier23, @NiuYawei, what are your opinions, please?

Reply:
I thought this is a sign of failure aggregation, then failed to reclaim space in some cases, no?

Reply:
ah, the container is in aggregation, and enter is for discard. ok, then maybe D_DEBUG. sorry for noise.

Reply:
OK. Since there's no new comments or concerns, I'll ask for landing...